MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -4732,6 +4732,30 @@ public void cancelMothballOrActivation() {
     }
     //endregion Mothballing/Activation
 
+    /**
+     * Returns a list of all soldiers in a unit.
+     *
+     * @return a list of Person objects representing the soldiers.
+     */
+    public List<Person> getSoldiers() {
```

Comment:
I suggest giving these methods good titles and/or comprehensive comments. We have many such methods in MM for equipment (hasWorkingMisc...) and it's never quite clear from title/comment what they really do. Might e.g. place the // comments into the javadoc. And, maybe state that soldiers are Infantry personnel (which includes BA)
Also, I believe in a case like this, the @return text alone would be alright. No need to repeat it as in l.4736 unless there's additional info somewhere.